src/views/v2/components/events-bar/search/submit.php

```diff
@@ -17,6 +17,7 @@
 	class="tribe-common-c-btn tribe-events-c-search__button"
 	type="submit"
 	name="submit-bar"
+	id="submit-bar-button"
```

Comment:
Is there a reason we need this ID here? If we have multiple instances of views on a page, we'll end up with 2 IDs that have the same value, which is not ideal.

Reply:
No - we can select based on the class.